### PR TITLE
Create atom.asc

### PR DESCRIPTION
**Pull request by G-T-P**
Atom is a fairly popular IDE that fully integrates with Git. This provides rudimentary explanation on how to set up Git integration with non github remote repositories from within a windows environment.
When I set up my environment, having access to these few lines would have saved me hours of googling...

### CODE REVIEW

File: book/A-git-in-other-environments/sections/atom.asc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+(((Atom)))
+Atom ships with a built-in package called GitHub, which provides a fairly-complete interface to Git operations and a deep integration with the website GitHub.
+There are different ways to access the package, probably the most common way is through the keybinding:
+
+Open the Git panel: `Ctrl+9`
+
+Another way is from the menu: `Packages -> GitHub -> Toggle Git Tab`
+
+GitHub comes with plenty of documentation when it comes to the integration with a remote repository on GitHub, which you can find within atom "Flight Manual".
```
Comment (HonkingGoose) [inline book/A-git-in-other-environments/sections/atom.asc]: Rationale for change:
- Atom is a noun and so needs to be capitalized. 
- Also we need a 's to indicate possession. Whose flight manual is it? It is Atom's Flight Manual.
---
Suggested situation after implementing changes:

GitHub comes with plenty of documentation when it comes to the integration with a remote repository on GitHub, which you can find within **Atom's** "Flight Manual".

File: book/A-git-in-other-environments/sections/atom.asc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+=== Clone repositories
+
+To clone a repository, access the command palette by typing `Ctrl + Alt + P` and then run the `GitHub: Clone` command.
+In the dialog paste a URL of a repository and click "Clone". A new project will get added to the Tree View.
+The URL to the remote repository for SSH connection should be of the form:
+
+    ssh://git@sub.domain.com:portNB/path/project.git
+
+For this operation to succeed, make sure that your git and your ssh are properly configured.
+Git is configured from a file called `.gitconfig` which must be located in your user directory (`%USERPROFILE%` on windows, run `git config` on Linux environments). Similarly SSH must be configured in a file `/.ssh/config` also located within your user directory on windows environement.
```
Comment (HonkingGoose) [inline book/A-git-in-other-environments/sections/atom.asc]: Rationale for change:
- The word "Windows" is a noun, it refers to a specific thing, so it needs to be capitalized.
- The spelling for the word environement is wrong, it is: environment.
---
Suggested situation after implementing changes:

For this operation to succeed, make sure that your git and your ssh are properly configured.
Git is configured from a file called `.gitconfig` which must be located in your user directory (`%USERPROFILE%` on **Windows**, run `git config` on Linux environments). Similarly SSH must be configured in a file `/.ssh/config` also located within your user directory on **Windows** **environment**.

**Comment by aollier** [comment]
I am not convinced of the relevance of this addition. All information needed for the GitHub package for Atom may be found at https://flight-manual.atom.io/using-atom/sections/github-package/, correct configuration of SSH is a classic and what is said about .gitconfig is not an information because it is something we already know.
Furthermore, @G-T-P deleted his/her repository so he/she will never do the requested changes.
So I suggest to close this Pull Request.
@ben Do you agree?

**Comment by G-T-P** [comment]
@aollier This PR was a starting point. The Appendix A: "Git in other environment" contains two big IDE (Eclipse and VS), with the growing popularity of Atom some extra info could be helpful to beginners. I know that as a git beginner myself I went looking for info on git-scm and I could not find it.
Having some of these basic information would have helped me at the time.

If it is not relevant then of course remove it :).

**Comment by aollier** [comment]
@G-T-P I think that if this information may be useful, its place is in Atom's or GitHub package's documentation.
If a system (Visual Studio, Eclipse, Atom, etc.) provides a way to interact with Git, it is the responsability of the maintainers of those systems to document it.

From my point of view, only parts [Graphical Interfaces](https://git-scm.com/book/en/v2/Appendix-A%3A-Git-in-Other-Environments-Graphical-Interfaces) and [Git in Bash](https://git-scm.com/book/en/v2/Appendix-A%3A-Git-in-Other-Environments-Git-in-Bash) have their place in the Appendix A.
Adding information about integration with other systems in https://git-scm.com duplicates information (as it should already be in the documentation of the system in question) and information in https://git-scm.com becomes regularly out-of-date as those systems progress.

For instance, the section about Visual Studio **is** already _out-of-date_ since the link takes us to Microsoft's "Azure Repos". I don't want to spend time to get the information in [Git in Visual Studio](https://git-scm.com/book/en/v2/Appendix-A%3A-Git-in-Other-Environments-Git-in-Visual-Studio) up-to-date since it may still progress and Microsoft already provides information to interact with Git.

**Comment by G-T-P** [comment]
> Adding information about integration with other systems in https://git-scm.com [...] becomes regularly out-of-date as those systems progress.

Even if this goes against my PR, this is a very valid point. Additionally it cannot be expected from git-scm to list all possible IDEs which include git natively or through plugins. 

In the end though the call is yours/@ben

**Comment by aollier** [comment]
Thanks anyway for help! 😄

**Comment by ben** [comment]
I wrestled with whether to accept this or not. The overall goal with including editor-specific content in the first place is that the book aims to be useful. It seemed likely 5 years ago that a user who is new to Git was likely to be working in something like Visual Studio or Eclipse. However, there's a long tail of other editors with Git integrations, and it's probable that the majority of users are _not_ using VS or Eclipse. Indeed, the [SO Survey last year](https://insights.stackoverflow.com/survey/2018/#development-environments-and-tools) found that Notepad++ and Vim were more popular than Eclipse.

I see an argument for including all editors who want to be here, since we want to be useful, but that's a ton of work. I also see an argument for removing all the editor-specific content altogether, since we don't want to be out-of-date. In the end, for the sake of cohesiveness and continuing quality, we have to be selective in which editors we have sections for. I'm going to close this PR, and set up an issue with the editors we _do_ want to support.

@G-T-P, thanks for your contribution. Even though we're not accepting these specific changes, you've contributed to making the book a better resource for the community!

**Pull request closed without merging**